src/ripple/app/ledger/impl/InboundLedger.cpp

```diff
@@ -981,9 +989,10 @@ bool InboundLedger::takeAsRootNode (Slice const& data, SHAMapAddNode& san)
         return false;
     }
 
-    AccountStateSF tFilter(app_);
+    AccountStateSF filter(mLedger->stateMap().family(),
```

Comment:
My poking around was leading in that direction. Thanks for confirming. The question is moot then.